pytorch_lightning/trainer/training_loop.py

```diff
@@ -180,7 +180,7 @@ class TrainerTrainLoopMixin(ABC):
     val_check_batch: ...
     num_val_batches: int
     disable_validation: bool
-    fast_dev_run: ...
+    unit_test: ...
```

Comment:
```suggestion
    unit_test: bool
```

Reply:
it's defined in trainer init, it should be inferred. I think keeping "..." is better.

Reply:
but it is always `bool` type, right? then we shall be as specific as we can...